app/views/layouts/_primary_navigation.html.haml

```diff
@@ -11,7 +11,8 @@
     - elsif current_user.persona.is_a?(CaseWorker)
       - if current_user.persona.admin?
         = link_to 'Summary', case_workers_admin_root_path
-        = link_to "Allocation", case_workers_admin_allocations_path
+        = link_to "Allocation", case_workers_admin_allocations_path(params.except(:controller, :action, :filter, :search).merge(tab: 'unallocated'))
```

Comment:
Do we want this link text to include the count of applicable claims (as the tabs did)?